### PR TITLE
Update VRS Schema in vrs_annotation_batch.py

### PR DESCRIPTION
**Pull request by matren395**
Added code from vrs_schema_renaming_script_temp.py to update the schema, added code to make assembly/referece_genome not static , TODO: add better versioning for different GnomAD versions , further productionize it for use on v4 !

### CODE REVIEW

File: vrs/vrs_annotation_batch.py [hunk 1]
```diff
@@ -246,15 +250,15 @@ def main(args):
         # Import all annotated shards
         ht_annotated = hl.import_vcf(
             annotated_file_list,
-            reference_genome="GRCh38",
+            reference_genome=assembly,
         ).make_table()
         logger.info("Annotated table constructed")
```
Comment (klaricch) [inline vrs/vrs_annotation_batch.py:256]: also need to add vrs_version to globals

File: vrs/vrs_annotation_batch.py [hunk 2]
```diff
@@ -246,15 +250,15 @@ def main(args):
...
 
         vrs_struct = hl.struct(
-            VRS_Allele=ht_annotated.info.VRS_Allele,
-            VRS_Start=ht_annotated.info.VRS_Start,
-            VRS_End=ht_annotated.info.VRS_End,
-            VRS_Alt=ht_annotated.info.VRS_Alt,
+            VRS_Allele_IDs=ht_annotated.info.VRS_Allele.split(","),
+            VRS_Starts=ht_annotated.info.VRS_Start.split(",").map(lambda x: hl.int(x)),
+            VRS_Ends=ht_annotated.info.VRS_End.split(",").map(lambda x: hl.int(x)),
+            VRS_States=ht_annotated.info.VRS_Alt.split(","),
```
Comment (klaricch) [inline vrs/vrs_annotation_batch.py:261]: does their updated code already output these with the correct names now, such as "VRS_Ends" instead of "VRS_End"?

**Review by klaricch — CHANGES_REQUESTED** [review]

File: vrs/vrs_annotation_batch.py (hunk 1 above)
Comment (klaricch) [reply vrs/vrs_annotation_batch.py:256]: reminder that vrs_version still needs to be added to the globals

**Review by klaricch — CHANGES_REQUESTED** [review]

File: vrs/vrs_annotation_batch.py [hunk 3]
```diff
@@ -246,15 +250,15 @@ def main(args):
...
         logger.info("Annotated table constructed")
 
         vrs_struct = hl.struct(
-            VRS_Allele=ht_annotated.info.VRS_Allele,
-            VRS_Start=ht_annotated.info.VRS_Start,
-            VRS_End=ht_annotated.info.VRS_End,
-            VRS_Alt=ht_annotated.info.VRS_Alt,
+            VRS_Allele_IDs=ht_annotated.info.VRS_Allele.split(","),
+            VRS_Starts=ht_annotated.info.VRS_Start.split(",").map(lambda x: hl.int(x)),
+            VRS_Ends=ht_annotated.info.VRS_End.split(",").map(lambda x: hl.int(x)),
```
Comment (matren395) [inline vrs/vrs_annotation_batch.py]:
```suggestion
            VRS_Starts=ht_annotated.info.VRS_Starts.split(",").map(lambda x: hl.int(x)),
            VRS_Ends=ht_annotated.info.VRS_Ends.split(",").map(lambda x: hl.int(x)),
```

File: vrs/vrs_annotation_batch.py (hunk 2 above)
Comment (matren395) [reply vrs/vrs_annotation_batch.py]: ype, the code in the most recent version does use 'VRS_Starts' and 'VRS_Ends', I'll be sure to include it in the next round of edits

**Comment by matren395** [comment]
07/10: Schema updated and runs without error and fills out VRS fields, plus added a global annotation. At Qin's request, moved it to gnomad_qc/gnomad_qc/v4/annotations as is (for testing on v4 exomes)

File: vrs/vrs_annotation_batch.py [hunk 4]
```diff
@@ -297,6 +300,11 @@ def main(args):
             f"gs://gnomad-vrs-io-finals/ht-outputs/annotated-checkpoint-VRS-{prefix}.ht"
         )
 
+        # Annotate final Hail Table with GA4GH version
+        ht_annotated = ht_annotated.annotate_globals(
+            ga4gh_vrs_version=ga4gh.vrs.__version__
```
Comment (klaricch) [inline vrs/vrs_annotation_batch.py:305]: in the quick fix and in the google doc of descriptions, this was just "vrs_version"

Comment (matren395) [reply vrs/vrs_annotation_batch.py:305]: corrected in commit coming in a few minutes

**Comment by matren395** [comment]
Note: should I be defining VRS_VERSION inside the "if annotate_original" branch if that is the only place it is used, or should I define it more higher up in the script? also changed the hardcoded reference to v3.1.2 in annotate_original

**Pull request merged**